Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/MSBuildExecution/MSBuildLocator.cs

```diff
@@ -30,14 +30,39 @@ namespace SonarScanner.MSBuild.Tasks.IntegrationTests
     /// </summary>
     internal static class MSBuildLocator
     {
+        /// <summary>
+        /// Returns a path to an instance of msbuild.exe or null if one could
+        /// not be found.
+        /// </summary>
+        /// <remarks>If there are multiple instances of VS on the machine there is no guarantee which
+        /// one will be returned, except that instances of VS2019 or later will be returned in preference
+        /// to VS2017.</remarks>
+        public static string GetMSBuildPath(TestContext testContext)
+        {
+            testContext.WriteLine($"Test setup: attempting to locate an MSBuild instance...");
+
+            var path = GetMSBuildPath("Current", testContext) // VS2019 or later
+                ?? GetMSBuildPath("15.0", testContext); // VS2017
```

Comment:
We'll now locate the latest version of MSBuild on the machine. Note that this will also find Preview versions of VS2019 or later, so it isn't guaranteed to be an RTM version.